src/lib/loaders/loaders_without_authentication/vortex.ts

```diff
@@ -0,0 +1,14 @@
+import factories from "../api"
+
+export default (opts) => {
```

Comment:
Ditto about default export. (We'll also want to be sure to use a full function name so that we get better stack traces)